scripts/tests/basic_test.py

```diff
@@ -0,0 +1,13 @@
+from ..wolf import test as wolftest
+
+snippet = r"""
+a = 'Hello'
+b = 3
+
+a
+b
+"""
+
+def test_basic(snapshot):
+    res = wolftest(snippet)
+    snapshot.assert_match(res)
```

Comment:
This is pretty cool, didn't know python had a library offering a snapshot feature like jest.